src/main/java/org/benetech/servicenet/service/impl/DocumentUploadServiceImpl.java

```diff
@@ -103,10 +103,14 @@ public boolean processFiles(final List<FileInfo> fileInfoList, final String prov
 
         fillLists(fileInfoList, parsedDocuments, documentUploads);
 
+        long startTime = System.currentTimeMillis();
+        log.info("Data upload for " + providerName + " has started");
         DataImportReport reportToSave = adapter
             .map(a -> a.importData(new MultipleImportData(parsedDocuments, documentUploads, report, providerName, true)))
             .orElse(report);
-
+        long stopTime = System.currentTimeMillis();
```

Comment:
please add TODO for removing this logic in the future, you cloud also add a link to an issue for refactoring the logs.